sycl/test/check_device_code/usm_pointers.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clangxx -fsycl-device-only -Xclang -fsycl-is-device -emit-llvm %s -S -o %t.ll -I %sycl_include -Wno-sycl-strict -Xclang -verify-ignore-unexpected=note,warning
+// RUN: FileCheck %s --input-file %t.ll
+//
+// Check the address space of the pointer in multi_ptr class
+//
+// CHECK: %[[DEVPTR_T:.*]] = type { i8 addrspace(5)* }
+// CHECK: %[[HOSTPTR_T:.*]] = type { i8 addrspace(6)* }
+//
+// CHECK-LABEL: define {{.*}} spir_func i8 addrspace(4)* @{{.*}}multi_ptr{{.*}}
+// CHECK: %m_Pointer = getelementptr inbounds %[[DEVPTR_T]]
+// CHECK-NEXT: %[[DEVLOAD:[0-9]+]] = load i8 addrspace(5)*, i8 addrspace(5)* addrspace(4)* %m_Pointer
+// CHECK-NEXT: %[[DEVCAST:[0-9]+]] = addrspacecast i8 addrspace(5)* %[[DEVLOAD]] to i8 addrspace(4)*
+// ret i8 addrspace(4)* %[[DEVCAST]]
+//
+// CHECK-LABEL: define {{.*}} spir_func i8 addrspace(4)* @{{.*}}multi_ptr{{.*}}
+// CHECK: %m_Pointer = getelementptr inbounds %[[HOSTPTR_T]]
```

Comment:
This test fails on the builds with disabled assertions. I suppose we should not check variable names - those are stripped.
https://github.com/intel/llvm/runs/834501651
Please, fix ASAP.

Reply:
Ok